site/content/en/latest/user/http-routing.md

```diff
@@ -9,17 +9,8 @@ Services. To learn more about HTTP routing, refer to the [Gateway API documentat
 
 ## Prerequisites
 
-Install Envoy Gateway:
-
-```shell
-helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace
-```
-
-Wait for Envoy Gateway to become available:
-
-```shell
-kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
-```
+Follow the steps from the [Quickstart](../quickstart) guide to install Envoy Gateway and the example manifest.
```

Comment:
can you confirm that this example still works ? want to make sure there are no hostname & path conflicts with the Quickstart httproute yaml

Reply:
I tested as above on my local machine and it works fine. 

However, to avoid documentation ambiguity, I think the Quick Start documentation could separate out the steps for installing envoy gateway so that it only installs eg. The steps for installing the resource experience can be put in a later article.

<img width="1082" alt="Snipaste_2024-02-16_17-21-16" src="https://github.com/envoyproxy/gateway/assets/77964041/dbce700b-95ab-47da-b16e-81c7050e9cef">

Reply:
imo a user should be able to quickly get familiar and run a demo using Quickstart instructions without having to navigate to another page

Reply:
> imo a user should be able to quickly get familiar and run a demo using Quickstart instructions without having to navigate to another page

yep, I think so too.